package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "graphql-subscription-server",
```

Comment:
should we scope this? `@4c/graphql-subscription-server`

Reply:
:+1: